source/GameInterface.Tests/Serialization/SerializerTests/ItemRosterSerializationTest.cs

```diff
@@ -0,0 +1,64 @@
+﻿using GameInterface.Serialization.Native;
+using GameInterface.Serialization;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+using TaleWorlds.CampaignSystem.Roster;
+using GameInterface.Serialization.Impl;
+using System.Reflection;
+using TaleWorlds.Core;
+using TaleWorlds.ObjectSystem;
+
+namespace GameInterface.Tests.Serialization.SerializerTests
+{
+    public class ItemRosterSerializationTest
+    {
+        [Fact]
+        public void ItemRoster_Serialize()
+        {
+            ItemRoster itemRoster = new ItemRoster();
+
+            BinaryPackageFactory factory = new BinaryPackageFactory();
+            ItemRosterBinaryPackage package = new ItemRosterBinaryPackage(itemRoster, factory);
+
+            package.Pack();
+
+            byte[] bytes = BinaryFormatterSerializer.Serialize(package);
+
+            Assert.NotEmpty(bytes);
+        }
+
+        [Fact]
+        public void ItemRoster_Full_Serialization()
+        {
+            MBObjectManager.Init();
```

Comment:
Create a constructor for EquipmentElementSerializationTest and move this line there and destroy it in the `Dispose` method